### PR TITLE
Support multiple example datasets

### PR DESCRIPTION
**Pull request by JovanVeljanoski**
With this PR, the `vaex.example(dataset)` command can support multiple datasets. 

At the moment the plan is to provide the following datasets:
- `dataset='astronomy'` - Helmi & De Zeeuw simulation data (default)
- `dataset='taxi'` - Sample of the NYC Yellow Cabs taxi data
- `dataset='iris'` - The Iris flower dataset (via `vaex.ml`)
- `dataset='titanic'` - The Titanic passenger list dataset (via `vaex.ml`)

### CODE REVIEW

File: packages/vaex-core/vaex/__init__.py [hunk 1]
```diff
@@ -579,12 +581,28 @@ def connect(url, **kwargs):
...
     """
-    return vaex.datasets.helmi_de_zeeuw_10percent.fetch()
+    if dataset == 'astronomy':
+        return vaex.datasets.helmi_de_zeeuw_10percent.fetch()
+    elif dataset == 'taxi':
+        return vaex.datasets.yellow_taxi_2012.fetch()
+    elif dataset == 'titanic':
+        return vaex.ml.datasets.load_titanic()
+    elif dataset == 'iris':
+        return vaex.ml.datasets.load_iris()
```
Comment (maartenbreddels) [inline packages/vaex-core/vaex/__init__.py:603]:
```suggestion
        import vaex.ml.datasets
        return vaex.ml.datasets.load_iris()
```

**Review by maartenbreddels — APPROVED** [review]
Was missing 2 imports, also a rebase should make this green.

File: packages/vaex-core/vaex/datasets.py [hunk 2]
```diff
@@ -45,7 +45,7 @@ def download_urlretrieve(self):
 
     def fetch(self, force_download=False):
         self.download(force=force_download)
-        return vx.open(self.filename)
+        return vaex.open(self.filename, convert=True)
```
Comment (maartenbreddels) [inline packages/vaex-core/vaex/datasets.py:48]: I guess this removes some of the metadata, which is causing the failure.

**Comment by JovanVeljanoski** [comment]
Following our discussion with @maartenbreddels  this will be refactored.

File: packages/vaex-core/vaex/__init__.py [hunk 3]
```diff
@@ -611,21 +613,8 @@ def connect(url, **kwargs):
...
-    return vaex.datasets.helmi_de_zeeuw_10percent.fetch()
-
-
-def zeldovich(dim=2, N=256, n=-2.5, t=None, scale=1, seed=None):
-    """Creates a zeldovich DataFrame.
-    """
-    import vaex.file
-    return vaex.file.other.Zeldovich(dim=dim, N=N, n=n, t=t, scale=scale)
-
+    return vaex.datasets.astronomy()
```
Comment (maartenbreddels) [inline packages/vaex-core/vaex/__init__.py:617]:
```suggestion
    return vaex.datasets.astronomy()
example.__doc__ = vaex.datasets.astronomy.__doc__
```

File: packages/vaex-core/vaex/datasets/__init__.py [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+    return _iris('iris_1e7', iris_1e6, int(1e7))
+
+
+def iris_1e8():
+    '''Returns the iris set repeated so it include ~1e8 rows'''
+    return _iris('iris_1e8', iris_1e7, int(1e8))
+
+
+def iris_1e9():
+    '''Returns the iris set repeated so it include ~1e8 rows'''
```
Comment (maartenbreddels) [inline packages/vaex-core/vaex/datasets/__init__.py:138]:
```suggestion
    '''Returns the iris set repeated so it include ~1e9 rows'''
```

**Review by maartenbreddels — COMMENTED** [review]
Love it, not sure what the error is about.

**Comment by maartenbreddels** [comment]
it's convert=True that gives the issue

**Comment by maartenbreddels** [comment]
Thanks @JovanVeljanoski !

**Pull request merged**